src/auth/useInteractiveLogin.ts

```diff
@@ -58,7 +58,7 @@ export function useInteractiveLogin(
             password,
           }),
         stateUpdated: (): void => {},
-        requestEmailToken: (): Promise<{ sid: string }> => {
+        requestEmailToken: async (): Promise<{ sid: string }> => {
           return Promise.resolve({ sid: "" });
         },
```

Comment:
This can be a one-liner
```suggestion
        requestEmailToken: async (): Promise<{ sid: string }> => Promise.resolve({ sid: "" }),
```